charts/osm/templates/osm-deployment.yaml

```diff
@@ -36,10 +36,10 @@ spec:
       {{- if not (.Capabilities.APIVersions.Has "security.openshift.io/v1") }}
       {{- include "restricted.securityContext" . | nindent 6 }}
       {{- end }}
-      initContainers:
-        - name: init-osm-controller
-          image: {{ .Values.osm.curlImage }}
-          command: ["curl", "http://osm-bootstrap.{{ include "osm.namespace" . }}.svc.cluster.local:9091/healthz", "--connect-timeout", "2", "--retry", "50", "--retry-connrefused", "--retry-delay", "5"]
+      # initContainers:
```

Comment:
this init container needs to stay, it's actually the part that checks to ensure the bootstrap pod has run successfully